docs/source/testing/testing.md

```diff
@@ -43,27 +72,76 @@ it('fetches single launch', async () => {
     { dataValues: { launchId: 1 } },
   ]);
 
-  // run query against the server and snapshot the output
+  // run the query against the server and snapshot the output
   const res = await server.executeOperation({ query: GET_LAUNCH, variables: { id: 1 } });
   expect(res).toMatchSnapshot();
 });
 ```
 
-This is an example of a full integration test being run against a test instance of `apollo-server`. This test imports the important pieces to test (`typeDefs`, `resolvers`, `dataSources`) and creates a new instance of `apollo-server`.
+The example above includes a test-specific [`context` function](../data/resolvers/#the-context-argument) which provides data directly to the `ApolloServer` instance, instead of calculating it from the request's context. 
 
-The example above shows writing a test-specific [`context` function](../data/resolvers/#the-context-argument) which provides data directly instead of calculating it from the request context. If you'd like to use your server's real `context` function, you can pass a second argument to `executeOperation` which will be passed to your `context` function as its argument. You will need to put together an object with the [middleware-specific context fields](../api/apollo-server/#middleware-specific-context-fields) yourself.
+If you'd like to use your server's real `context` function, you can pass a second argument into `executeOperation` which is then passed to your server's `context` function. Note that in order to use your server's `context` function you need to put together an object with the correct [middleware-specific context fields](../api/apollo-server/#middleware-specific-context-fields) for your implementation.
```

Comment:
"If you'd like to use your server's real `context` function" -> "To use your server's defined `context` function"

"argument into" -> "argument to" (I think passing arguments _to_ a function is sufficient) 

"Note that in order to" -> "Note that to" ("in order to" can almost always be shortened to "to")